client/buttonPress.py

```diff
@@ -0,0 +1,19 @@
+import pygame as pg
+from definitions import *
+
+class Button(pg.Rect):
+    def __init__(self, w, h, text, position):
+        pg.Rect.__init__(self, *position, w, h)
+        self.font = pg.font.Font(asset('CaviarDreams.ttf'), 18)
+        self.text = self.font.render(text, True, BLACK)
+        self.color_inactive = BANANA
```

Comment:
read comment about active/inactive colors in the textbox; same point applies here